waspc/packages/deploy/src/providers/fly/IGlobalOptions.ts

```diff
@@ -0,0 +1,4 @@
+export interface IGlobalOptions {
```

Comment:
Good to know, I can remove them. Read one doc/site where they preferred it, but I don't personally either... It was giving me C# vibes :D

Reply:
Skipping the `I` is a good decision, here's a quote from the official TypeScript handbook:
> In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.

Also, [here's the reasoning](https://stackoverflow.com/a/41967120) that convinced me back in the day.

Reply:
Good points in the quote and link! I changed the filenames but forgot to update the actual type names 😅 will do that shortly (as part of one other refactor). Thanks all.